projects/llvm/build.sh

```diff
@@ -39,6 +39,13 @@ else
     llvm-special-case-list-fuzzer \
   )
 fi
+# Fuzzers whose inputs are C-family source can use clang-fuzzer-dictionary.
+readonly CLANG_DICT_FUZZERS=( \
+  clang-fuzzer \
+  clang-format-fuzzer \
+  clang-objc-fuzzer \
+  clang-pseudo-fuzzer \
```

Comment:
clang-pseudo-fuzzer isn't running yet, but this PR should merge cleanly with #7679 and work fine either order.